tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.spec.js

```diff
@@ -265,8 +265,8 @@ describe('Feature: Repeating Sections with Hub and Spoke', function () {
       $(VisitorsDateOfBirthPage.year()).setValue('1999');
       $(VisitorsDateOfBirthPage.submit()).click();
 
-      expect($(HubPage.submit()).getText()).to.equal('Submit survey');
-      expect($(HubPage.displayedName()).getText()).to.equal('Submit survey');
+      expect($(HubPage.submit()).getText()).to.equal('Submission text');
+      expect($(HubPage.displayedName()).getText()).to.equal('Title displayed on hub when complete');
```

Comment:
Should these have changed? I can't see any changes to the `test_repeating_sections_with_hub_and_spoke.json` schema.